src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -196,11 +240,81 @@ public void LogWarning(Runtime.WarningID ID, string message, string filename, in
             }
         }
 
+        internal void LogInfo(Runtime.InfoID ID, string message, string filename, int line, int col)
+        {
+            filename ??= string.Empty;
+
+            if (!runtimeCore.Options.IsDeltaExecution && (string.IsNullOrEmpty(filename) ||
+                line == Constants.kInvalidIndex ||
+                col == Constants.kInvalidIndex))
+            {
+                AuditCodeLocation(ref filename, ref line, ref col);
+            }
+
+            var infoMsg = string.Format(Resources.kConsoleWarningMessage,
+                                           message, filename, line, col);
+
+#if DEBUG
+            if (runtimeCore.Options.Verbose)
+            {
+                Console.WriteLine(infoMsg);
```

Comment:
@aparajit-pratap @pinzart90 @sm6srw @mjkkirschner Do you have a suggestion on if writing info would be as valuable as writing runtime warnings?

Reply:
I end up removing the logging since we may not care about graph info as much as a warning. We can add it back if we care